src/frontend/packages/lib_classroom/src/data/classroomAppData.ts

```diff
@@ -0,0 +1,5 @@
+import { parseDataElements } from 'utils/parseDataElements/parseDataElements';
+
+export const classroomAppData = parseDataElements(
```

Comment:
I think this should be kept in the lti_site. Only in the `lti_site` we parse the DOM to retrieve data information related to the current request. This will not happen in the standalone site.

Reply:
And so the `parseDataElements` should be in the `lti_site`.

And so probably, component using directly `classroomAppData` should be refacto to not use it directly anymore.

Reply:
It should be ok on the last commit "[♻️(lib-classroom) remove classroomAppData](https://github.com/openfun/marsha/pull/1783/commits/75c9e00c7b330f1c3835f8f1d117c6762a1e84fe)", I have a small doubt on `src/frontend/packages/lib_classroom/src/components/DashboardClassroomInstructor/index.tsx`